docs/std-seq.md

```diff
@@ -90,3 +92,75 @@ Usage:
 | `//seq.split(" ", "deliberately adding spaces to demonstrate the split function")` | `["deliberately", "adding", "spaces", "to", "demonstrate", "the", "split", "function"]` |
 | `//seq.split("random stuff", "this is just a random sentence")` | `["this is just a random sentence"]` |
 
+## Array
```

Comment:
The behaviour is exactly the same across strings and array. You should document them once and merge the examples.